tests/component/test_trigger_properties.py

```diff
@@ -0,0 +1,136 @@
+"""Contains a collection of pytest tests that validates trigger properties."""
+
+import nidaqmx
+from nidaqmx.constants import TriggerType
+from nidaqmx.error_codes import DAQmxErrors
+from nidaqmx.errors import DaqError
+
+
+def test__trigger__get_float_property__returns_value(any_x_series_device):
+    """Test to validate getter for trigger property of float type."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+
+        # Default Rate of the pulse width filter timebase in NI PCIe-6363 device is 0.0
+        assert task.triggers.start_trigger.dig_edge_dig_fltr_timebase_rate == 0.0
+
+
+def test__trigger__set_float_property__returns_assigned_value(any_x_series_device):
+    """Test to validate setter for trigger property of float type."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+
+        value_to_test = 2.505
+        task.triggers.start_trigger.dig_edge_dig_fltr_timebase_rate = value_to_test
+        assert task.triggers.start_trigger.dig_edge_dig_fltr_timebase_rate == value_to_test
+
+
+def test__trigger__reset_float_property__returns_initial_value(any_x_series_device):
+    """Test to validate resetting trigger property of float type."""
+    with nidaqmx.Task() as task:
+        task.ai_channels.add_ai_voltage_chan(any_x_series_device.ai_physical_chans[0].name)
+
+        # Default Rate of the pulse width filter timebase in NI PCIe-6363 device is 0.0
+        del task.triggers.start_trigger.dig_edge_dig_fltr_timebase_rate
```

Comment:
this test currently could be passing if reset is a noop - set to non-default (you don't need to validate that worked since another test does that) before reset

Reply:
Resolved. Set non-default value before resetting.